### PR TITLE
Life insurance text

### PR DESCRIPTION
**Pull request by cvalarida**
Connected to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3837

I wasn't super sure if that ticket was talking about the `benefits/plot-allowance` page, `amountGovtContribution` field, but that's what I changed.

image

### CODE REVIEW

**Comment by jbalboni** [comment]
@cvalarida Sorry about that, I believe it was the other field.

**Comment by cvalarida** [comment]
Where "the other field" = `govtContributions`?

**Comment by jbalboni** [comment]
@cvalarida Yes.

**Comment by cvalarida** [comment]
Updated. Thanks!

**Review by cehsu — APPROVED** [review]
Looks good to me.

File: src/js/burials/config/form.js [hunk 1]
```diff
@@ -182,6 +183,20 @@ const formConfig = {
           uiSchema: {
             deathDate: currentOrPastDateUI('Date of death'),
             burialDate: currentOrPastDateUI('Date of burial (includes cremation or interment)'),
+            'view:burialDateWarning': {
+              'ui:description': burialDateWarning,
+              'ui:options': {
+                hideIf: formData => {
+                  // If they haven't entered a complete year, don't jump the gun and show the warning
+                  if (formData.burialDate && !/\d{4}-\d{1,2}-\d{1,2}/.test(formData.burialDate)) {
```
Comment (jbalboni) [inline src/js/burials/config/form.js]: Did this get included from another PR?

Comment (cvalarida) [inline src/js/burials/config/form.js]: Boo, yes.

**Comment by cvalarida** [comment]
Ugh, I built this on top of the other branch I was working on. Humbug. Messed the history up enough it'll be easier to just start a new branch, so I'm doing that. 😞

Meh.

**Pull request closed without merging**